oe_paragraphs.post_update.php

```diff
@@ -200,3 +201,12 @@ function oe_paragraphs_post_update_10006(array &$sandbox): void {
   $field->set('settings', $settings);
   $field->save();
 }
+
+/**
+ * Change the description of oe_paragraphs_description_list paragraph.
+ */
+function oe_paragraphs_post_update_10008(array &$sandbox) {
```

Comment:
```suggestion
function oe_paragraphs_post_update_10007(array &$sandbox): void {
```
I know there is another oe_paragraphs_post_update_10007 in a different PR, but until we don't know which one will be merged first we have to put here the next iteration